app/src/components/widgets/charts/investigatedTrendChart/investigatedTrendChart.jsx

```diff
@@ -72,6 +77,8 @@ export class InvestigatedTrendChart extends Component {
 
   static defaultProps = {
     navigate: () => {},
+    getDefectLink: () => {},
+    getStatisticsLink: () => {},
```

Comment:
1. looks like it's an object (see line 68: `getStatisticsLink: PropTypes.object.isRequired`)
2. If it's required, you don't need default props